config/deploy/delayed_job_statsd.yaml.erb

```diff
@@ -0,0 +1,52 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: delayed-jobs-statsd
+  annotations:
+    shipit.shopify.io/restart: 'true'
+spec:
+  selector:
+    matchLabels:
+      name: delayed-jobs-statsd
+  template:
+    metadata:
+      labels:
+        name: delayed-jobs-statsd
+    spec:
+      containers:
+      - name: delayed-jobs-statsd
+        image: dwradcliffe/delayed_job_statsd@sha256:cf94a0a92e794b4ae9a7a9dbc87e4472f5d38e066a1c5bc3ba123e7ad576fc6f
```

Comment:
`rubygems/rubygems.org` is on `quay.io`

Reply:
Yeah, I'll move this over at some point.